cadvisor.go

```diff
@@ -58,6 +59,8 @@ var enableProfiling = flag.Bool("profiling", false, "Enable profiling via web in
 var collectorCert = flag.String("collector_cert", "", "Collector's certificate, exposed to endpoints for certificate based authentication.")
 var collectorKey = flag.String("collector_key", "", "Key for the collector's certificate")
 
+var dockerLabels = flag.Bool("docker_labels", true, "convert docker labels and environment variables into labels on prometheus metrics for each docker container. If flag set to false, then only metrics exported are container name, first alias, and image name")
```

Comment:
This isn't limited to docker labels (as it works for other container runtimes that expose labels), lets rename this to `store_container_labels` or something similar.